data/human/hails.txt

```diff
@@ -1708,6 +1710,77 @@ phrase "friendly civilian"
 		", but I still had to take a big loan to buy my first ship."
 		", but it was my grandma's inheritance that allowed me to buy this ship."
 
+phrase "friendly civilian"
+	word
+		"Don't be too picky about the jobs you take. It is better to"
+	word
+		" "
+	word
+		" ship crooks, than having a crooked ship."
+		" ship shells, than having your ship shelled."
+		" ship slugs, than having a sluggish ship."
+		" ship junk, than having a junk ship."
+		" ship buckles, than having your ship bucle."
+		" ship squash, than having your ship squashed."
+		" ship bad bands, than having your ship badly banned."
+		" ship shots, than having your ship shot."
+		" be working hard at shipping, than having your ship hardly working."
+		" fly to the ruins, than flying in ruins."
+		" be fired, than having your ship on fire."
+		" transport mines, than handling transport in the mines."
+
+
+phrase "friendly civilian"
```

Comment:
Is this captain supposed to seem drunk? Same complaint.

Reply:
I was trying to create this type of joke: https://en.wikipedia.org/wiki/In_Soviet_Russia, but maybe I didn't do it correctly.

Reply:
If I read them with a stereotypical Russian accent, it makes a bit more sense, but of course there's no way to communicate this to the player. :) I just feel like these are a bit too meme-y. But maybe I'm just being an old fart. :)

Reply:
They are a bit meme-y, but it's just 4 possible hails out of the large set of civilian hails we have.